src/store/src/Bridge/ClickHouse/Store.php

```diff
@@ -25,12 +25,12 @@
 /**
  * @author Grégoire Pineau <lyrixx@lyrixx.info>
  */
-class Store implements ManagedStoreInterface, StoreInterface
+final readonly class Store implements ManagedStoreInterface, StoreInterface
```

Comment:
Are the other stores final? I would not make it final, as Symfony is in general more open

Reply:
Yes, all of them are `final` except this one.